web/frontend/src/pages/form/GroupedResult.tsx

```diff
@@ -124,9 +124,15 @@ const GroupedResult: FC<GroupedResultProps> = ({ rankResult, selectResult, textR
           const select = element as SelectQuestion;
 
           if (selectResult.has(id)) {
-            res = countSelectResult(selectResult.get(id)).resultsInPercent.map((percent, index) => {
-              return { Candidate: select.Choices[index], Percentage: `${percent}%` };
-            });
+            res = countSelectResult(selectResult.get(id))
+              .map(([, totalCount], index) => {
+                return {
+                  Candidate: select.Choices[index],
+                  TotalCount: totalCount,
+                  NumberOfBallots: selectResult.get(id).length,
```

Comment:
Question: what is `NumberOfBallots` here?
- the total ballots cast (including multi ballot cast)
- the total number of voters (with multi ballot cast removed)
- the total number of valid ballots (with multi ballot and invalid ballots removed)

I think both 2 and 3 are OK, but it should be mentioned somewhere on the page which it is.

Reply:
it's the number of valid ballots for the 'select' type of form as send back by the server